File: Domovoy/Models/Informer/Informer.cs
namespace Domovoy.Models;

public class Informer
{
    public int Id { get; set; }

    public string Name { get; set; }
    public DateTime LastInform { get; set; }
    // Is working
    public bool IsActive { get; set; }

    public Apartment Apartment { get; set; }
    public int ApartmentId { get; set; }

    public int? InformMeterId { get; set; }
    public InformMeter? InformMeter { get; set; }
    public int? EventInformerId { get; set; }
    public EventInformer? EventInformer { get; set; }
}

File: Domovoy/ClientApp/src/api/models/Informer.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { Apartment } from './Apartment';
import type { EventInformer } from './EventInformer';
import type { InformMeter } from './InformMeter';

export type Informer = {
    id?: number;
    name?: string | null;
    lastInform?: string;
    isActive?: boolean;
    apartment?: Apartment;
    apartmentId?: number;
    informMeterId?: number | null;
    informMeter?: InformMeter;
    eventInformerId?: number | null;
    eventInformer?: EventInformer;
};